libpod/container.go

```diff
@@ -884,3 +888,8 @@ func (c *Container) cleanupStorage() error {
 
 	return nil
 }
+
+// CGroupPath returns a cgroups "path" for a given container.
+func (c *Container) CGroupPath() cgroups.Path {
+	return cgroups.StaticPath(filepath.Join(CGroupParent, fmt.Sprintf("libpod-conmon-%s", c.ID())))
```

Comment:
In CRI-O they use two different Cgroups one is libpod-conmon-CID and the other is just libpod-CID